src/NuGet.Clients/NuGet.VisualStudio.Common/OutputConsoleLogger.cs

```diff
@@ -67,7 +67,7 @@ public class OutputConsoleLogger : INuGetUILogger, IDisposable
                 _buildEvents.OnBuildBegin += (_, __) => { ErrorListTableDataSource.Value.ClearNuGetEntries(); };
                 _solutionEvents = _dte.Events.SolutionEvents;
                 _solutionEvents.AfterClosing += () => { ErrorListTableDataSource.Value.ClearNuGetEntries(); };
-                OutputConsole = await consoleProvider.CreatePackageManagerConsoleAsync();
+                return await consoleProvider.CreatePackageManagerConsoleAsync();
```

Comment:
It might be a good idea to pass the JoinableTaskFactory as an extra param to the `AsyncLazy`, just in case someone tries to evaluate it synchronously.

Reply:
I do not think we need to do that -- we are accessing it from global singleton.

Reply:
Every call is wrapped in a JTF.Run, so it the context is passed. 
Resolving.